mtags/src/main/scala-2/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -454,26 +454,38 @@ trait MatchCaseCompletions { this: MetalsGlobal =>
         else pattern
       val cursorSuffix = (if (patternOnly.nonEmpty) "" else " ") +
         (if (clientSupportsSnippets) "$0" else "")
-      new TextEditMember(
-        filterText = label,
-        edit = new l.TextEdit(
-          editRange,
-          label + cursorSuffix
-        ),
-        sym = sym,
-        label = Some(label),
-        additionalTextEdits = autoImports
+
+      val edit = new l.TextEdit(
+        editRange,
+        label + cursorSuffix
       )
+      if (patternOnly.nonEmpty && name == pattern) {
```

Comment:
This is a case to ensure that in something like

```scala
Option(123) match {
  case No@@
}
```

you only see `None` once instead of once for something from match completions then again from the general scope completions.